hoodie-client/src/main/java/com/uber/hoodie/io/HoodieAppendHandle.java

```diff
@@ -103,7 +103,7 @@ private void init(Iterator<HoodieRecord<T>> recordItr) {
                             + " on commit " + commitTime + " on HDFS path " + hoodieTable
                             .getMetaClient().getBasePath() + partitionPath, e);
                 }
-                writeStatus.getStat().setFullPath(currentLogFile.getPath().toString());
+                writeStatus.getStat().setRelativePath(partitionPath);
```

Comment:
I dont think this is correct. The relative path is the substring of currentLogFile.getPath() where basePath ends. PartitionPath is just the relative partition path and not the file path.